.env

```
@@ -8,3 +8,5 @@ EMOTION_MODEL_NAME=state-spaces/mamba-130m-hf
 EMOTION_FP16=false
 EMOTION_BF16=false
 MAX_SEQUENCE_LENGTH=4096
+SHIVA_SCRATCHPAD_DB=scratchpad.db
+SHIVA_MEMORY_DB=memory.db
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SHIVA_MEMORY_DB key should go before the SHIVA_SCRATCHPAD_DB key

(UnorderedKey)